### DISCUSSION TITLE
Recording MCAP through Studio's UI

### DISCUSSION THREAD

**Discussion by WaelKarkoub** [foxglove/community#34] [Q&A]
In the scenario where you're monitoring a robot live, it can be useful to click a button that starts recording when you observe an event (maybe it's always recording in the background and only saves the last n-number of minutes?), which then saves the mcap file locally or pushes it to the Data Platform.

**Comment by jtbandes** [comment]
We have a ticket for this request at https://github.com/foxglove/studio/issues/786
